api/definitions/v1.buildconfigstatus/description.adoc

```diff
@@ -0,0 +1 @@
+A BuildConfigStatus contains current state of the <<v1.BuildConfig>> object. The status is stored as an integer andstores the number of the last triggered build.
```

Comment:
contains *the* current state...

> The status is stored as an integer andstores the number of the last triggered build.

The status contains the number of the last triggered build.